Annex60/Fluid/Actuators/BaseClasses/PartialTwoWayValve.mo

```diff
@@ -19,6 +19,9 @@ partial model PartialTwoWayValve "Partial model for a two way valve"
   input Real phi
     "Ratio actual to nominal mass flow rate of valve, phi=Kv(y)/Kv(y=1)";
 protected
+ constant Boolean assertPhiPos = true
```

Comment:
I suggest to remove this as its value can only be changed by models that extend this model, but not by a user because it is protected.
Also, `Evaluate=true` is not needed for `constant` as constants are evaluated at compile time.